### DISCUSSION TITLE
Vue can be moved to devDependencies

### DISCUSSION THREAD

**Discussion by raphaelmenges** [cawa-93/vite-electron-builder#897] [Q&A] [answered]
Hi 👋

Currently, `vue` is listed under `dependencies` in the [package.json](https://github.com/cawa-93/vite-electron-builder/blob/4a788e5bea335155d9b1df86591fdd40097df950/package.json#L54). I argue it might and should be moved to `devDependencies`, because vite and rollup are including any required code of the renderer process into the bundle already. To my experience, only packages that are used by the main or the preload process need to be listed under `dependencies`, i.e., because `ssr: true` is set in the `vite.config.js`. This also probably means that a module used in both the renderer process and the main or preload process is shipped twice, once bundled and once as stand-alone module?

**Comment by cawa-93** [comment, marked as answer]
Good point. PR welcome
